src/biz/metrics_test.go

```diff
@@ -159,7 +150,7 @@ func stress(t *testing.T, args ...string) (containerID string, closeFunc func())
 	t.Helper()
 
 	arguments := []string{
-		"run", "-d", "--rm",
```

Comment:
`--rm` is not needed  and this function returns a "cleanup" function that is used to remove the container after it's not needed or the test finishes.